src/scopes/goals/topics.js

```diff
@@ -33,7 +32,13 @@ export function withTopics(topics, options) {
   return {
```

Comment:
We cannot use 'ARRAY_TO_STRING(ar."topics", ',')`;' here unless you include % % around the topic name (but it wont be accurate.

For example: 

ARRAY_TO_STRING(ar."topics", ',') ILIKE 'Communication' 

wont work but this would: ARRAY_TO_STRING(ar."topics", ',') ILIKE '%Communication%'

Probably better off doing this:  ar."topics" && '{"Communication"}'




B

Reply:
ok I think this is all set